var/spack/repos/builtin/packages/elpa/package.py

```diff
@@ -76,7 +75,14 @@ def setup_environment(self, spack_env, run_env):
         spack_env.set('SCALAPACK_LDFLAGS', spec['scalapack'].libs.joined())
```

Comment:
btw, @alalazo why did you move those from being options to `configure` here?

Reply:
To group together all the changes to the environment variables in the function that is meant to change the environment variables for this package.